src/SortableList.js

```diff
@@ -17,8 +17,8 @@ export default class SortableList extends Component {
   static propTypes = {
     data: PropTypes.object.isRequired,
     order: PropTypes.arrayOf(PropTypes.any),
-    style: View.propTypes.style,
-    contentContainerStyle: View.propTypes.style,
+    style: PropTypes.any,
+    contentContainerStyle: PropTypes.any,
```

Comment:
using PropTypes.any is bad practice imo, lets use ViewPropTypes from react-native